xinference/model/llm/ggml/llamacpp.py

```diff
@@ -30,34 +32,9 @@
 from ..utils import ChatModelMixin
 from .ctransformers import CTRANSFORMERS_SUPPORTED_MODEL
 
-if TYPE_CHECKING:
-    from llama_cpp import LogitsProcessorList, StoppingCriteriaList
-
 logger = logging.getLogger(__name__)
 
 
-class LlamaCppGenerateConfig(TypedDict, total=False):
```

Comment:
Let the server decide what the generate config should be. It is OK to have a duplicated definition on the client side.